gcsweb/cmd/gcsweb/gcsweb.go

```diff
@@ -127,14 +131,52 @@ func gatherOptions() options {
 
 func (o *options) validate() error {
 	// validate and parse bucket list
+	o.bucketAliases = make([]*bucketAlias, 0)
 	for _, bucket := range o.allowedBuckets {
+		bucketParts := strings.Split(bucket, "=")
```

Comment:
Can we get a function `func parseBucket(bucket string) (name string, aliases []*bucketAlias, error)` and some unit tests for the input?

Reply:
I've refactored the code out into its own function but I found the signature `func (o *options) parseBucket(bucket string) error` to be a better fit. The function needs to fill both `o.allowedProwPaths` and `o.bucketAliases` so it's just more convenient doing everything in there. Whenever we find a new use case I'll refactor even more, what do you think about it?